Csip/struct map/csip81

ltaimre opened the pull request.

ltaimre committed:
Updated testCase.xml

ltaimre committed:
Merge branch 'integration' into csip/structMap/CSIP81

PhillipTommerholt committed:
Update testCase.xml

koit committed:
Merge branch 'integration' into csip/structMap/CSIP81

koit commented:
#152

koit reviewed (dismissed):
Reviewed corpora/csip/metadata/structmap/CSIP81/testCase.xml - it is OK. All the other files are irrelevant and should not be in this PR.

koit committed:
Update CSIP80/testCase.xml from integration

This is to solve an annoying conflict - due to mistakes in branch
creation, some files ended up in wrong branches, in this case
CSIP80/testCase.xml was in CSIP81 branch and got into conflict with the
newer, correct instance of that file in integration.

koit committed:
Merge branch 'integration' into csip/structMap/CSIP81
